Frameworks/Foundation/NSURLSessionTask.mm

```diff
@@ -219,23 +219,18 @@ - (void)_stopLoading {
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
```

Comment:
@rajsesh-msft there do not appear to be IHttpClient et al subclasses that can offer this to us. I was able to find the test page we talked about in the Foundation-Dev test app, but it only has Cancel/Resume and not suspend.